packages/ag-charts-community/src/chart/interaction/toolbarManager.ts

```diff
@@ -44,19 +45,20 @@ export interface ToolbarFloatingAnchorChangedEvent extends Event<'floating-ancho
 }
 
 export interface ToolbarButtonPressedEvent<T = any> extends Event<'button-pressed'> {
+    id: string;
     value: T;
     rect: BBox;
 }
 
-export interface ToolbarButtonToggledEvent<T = any> extends Event<'button-toggled'> {
-    value: T;
+export interface ToolbarButtonToggledEvent extends Event<'button-toggled'> {
```

Comment:
Can we keep this `value: T` typing? It is used in annotations https://github.com/ag-grid/ag-charts/blob/latest/packages/ag-charts-enterprise/src/features/annotations/annotations.ts#L358

Reply:
ignore this, sorry